.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3.0.2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: 8
+          cache: 'sbt'
+      - name: npm install
+        run: npm install
```

Comment:
Why is this needed?

Reply:
We also use in the `ci.yml`, so I put it here for consistency of the environment.

But in theory it shouldn't be needed. Do you prefer if I remove it?

Reply:
I think it is better to omit it in this case (to avoid unnecessary computations)

Reply:
OK, I removed it.